routes/api/account.js

```diff
@@ -173,6 +174,37 @@ module.exports = {
             Controllers.Account.getUserById
         );
 
+        /**
+         * @api {patch} /account/confirmation/:id update an account's confirmed status
+         * @apiName patchAccountConfirmed
+         * @apiGroup Account
+         * @apiVersion 0.0.9
+         * 
+         * @apiParam (body) {Boolean} [confirmed] Whether the account is confirmed or not
+         * @apiSuccess {string} message Success message
+         * @apiSuccess {object} data Hacker object
+         * @apiSuccessExample {object} Success-Response: 
+         *      {
+         *          "message": "Changed account information", 
+         *          "data": {
+         *              "confirmed": "true"
```

Comment:
Change `"true"` to `true` to represent boolean value

Reply:
Should someone be able to change their own account from confirmed to not confirmed?

Reply:
Now that I think of it, there really only should be one way of confirming an account-- via the auth confirm route: <https://hackerapi.mchacks.ca/api/#api-Authentication-confirmAccount>

Reply:
True, removed route